### PR TITLE
feat(onyx-718): add partnerOffer field for PartnerOfferCreated notifications

### PR DESCRIPTION
**Pull request by nickskalkin**
Follow-up for https://github.com/artsy/gravity/pull/17314

We need to display price with and without discount in activity panel for partner offer notifications.

### CODE REVIEW

**Comment by nickskalkin** [comment]
Changing Float to Money..

**Comment by nickskalkin** [comment]
Moving to draft once again. I want to format price message on MP side

File: src/schema/v2/partnerOffer.ts [hunk 1]
```diff
@@ -16,6 +18,9 @@ export const PartnerOfferType = new GraphQLObjectType<any, ResolverContext>({
       type: GraphQLString,
       resolve: ({ artwork_id }) => artwork_id,
     },
+    available: {
+      type: GraphQLBoolean,
+    },
```
Comment (anandaroop) [inline src/schema/v2/partnerOffer.ts:23]: **suggestion (non-blocking):** Minor, but I believe we prefer to rename these upstream booleans to e.g. `isAvailable`

(Examples [here](https://github.com/artsy/metaphysics/blob/836358a9fc9b960ee18ab7ec606ed331bbe488bf/src/schema/v2/me/index.ts#L388-L391) and [here](https://github.com/artsy/metaphysics/blob/836358a9fc9b960ee18ab7ec606ed331bbe488bf/src/schema/v2/gene.ts#L137-L140))

**Review by anandaroop — DISMISSED** [review]
👍🏽

File: src/schema/v2/partnerOffer.ts (hunk 1 above)
Comment (nickskalkin) [reply src/schema/v2/partnerOffer.ts]: thanks, fixed!

File: src/schema/v2/partnerOffer.ts [hunk 2]
```diff
@@ -25,6 +31,28 @@ export const PartnerOfferType = new GraphQLObjectType<any, ResolverContext>({
       type: GraphQLString,
       resolve: ({ partner_id }) => partner_id,
     },
+    priceListedMessage: {
+      type: GraphQLString,
```
Comment (MounirDhahri) [inline src/schema/v2/partnerOffer.ts:35]: I see you initially went with this type as a `Money` interface then changed your mind. Is there a specific reason for that?

Comment (nickskalkin) [reply src/schema/v2/partnerOffer.ts:35]: @MounirDhahri sure! I've just noticed that Artwork details component uses `saleMessage` field from Metaphysics, so I've decided to stick to the same approach. `saleMessage` is a formatted string 

- saleMessage field resolver in MP: https://github.com/artsy/metaphysics/blob/ec13f424f9547d2961f2d71a6ff1f920a14ebbb9/src/schema/v2/artwork/index.ts#L1501
- place where gravity queries it: https://github.com/artsy/force/blob/main/src/Components/Artwork/Details.tsx#L376

Comment (nickskalkin) [reply src/schema/v2/partnerOffer.ts:35]: You think it's worth re-doing it with Money?

**Pull request merged**